print_list.js

```diff
@@ -0,0 +1,34 @@
+
+class Node {
+  constructor(value, next) {
+    this.value = value;
+    this.next = next;
+  }
+
+  getNext() {
+    return this.next;
+  }
+
+  setNext(next) {
+    this.next = next;
+  }
+
+  getValue() {
+    return this.value;
+  }
+
+  setValue(value) {
+    this.value = value;
+  }
+
+  printList() {
+    let current = this;
+
+    while (current != null) {
+      console.log(current.value);
+      current = current.getNext();
+    }
+  }
```

Comment:
It may be better to convert this into a `toString()` method and have console.log() called on Node outside of the class in case of any unwanted print messages.

Reply:
Consider: Perhaps a function to printList should probably live in the list class (not in the node).